resources/views/myhub/company/details.blade.php

```diff
@@ -25,7 +25,7 @@
                         @livewire('company.manage-logo', ['company' => $company])
```

Comment:
When I am a company representative and the company is approved I am unable to edit the company logo
<img width="629" alt="image" src="https://github.com/HZ-HBO-ICT/it-conference/assets/77020646/5c08f560-2a6d-4c08-981d-7a417fdc6603">

Reply:
Fixed in [21cc775](https://github.com/HZ-HBO-ICT/it-conference/pull/411/commits/21cc7752c176550b619e7452da865ba6368d5446)